src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanEnterpriseBuildStep.java

```diff
@@ -470,6 +471,16 @@ private void shouldFailBuild(IResultsProvider provider, Run<?, ?> build) throws
 		}
 	}
 
+    	private boolean checkURLAccessibility(String URL) throws IOException {
+        	try {
+            		URL url = new URL(URL);
+            		HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+            		return conn.getResponseCode() == HttpURLConnection.HTTP_OK;
```

Comment:
Do we only want to pass this test if the response code is explicitly 200 or could it be any 2XX code (e.g. 202)?

Reply:
Matt, I think for this specific case "200" would be fine.